### PR TITLE
install.md: building netCDF with PnetCDF support

### PR DESCRIPTION
**Pull request by wkliao**
add missing info about building netCDF with PnetCDF support

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -78,6 +78,8 @@ Requirements {#netcdf_requirements}
   * HDF5 1.10.1 or later.
 * zlib 1.2.5 or later (for netCDF-4 compression)
 * curl 7.18.0 or later (for DAP remote access client support)
+* For paralle I/O support on classic netCDF files
+  * PnetCDF 1.6.0 or later
```
Comment (edhartnett) [inline docs/install.md]: I test on 1.8.0 and 1.9.0, I don't know what Ward tests with. 1.6.0 seems pretty downrev for pnetcdf.

File: docs/install.md [hunk 2]
```diff
@@ -271,19 +273,33 @@ From the top-level netCDF-4 source directory, the following builds netCDF-4 with
...
+    $ # Build and install PnetCDF
+    $ PNDIR=/usr/local
+    $ ./configure --prefix=${PNDIR} --with-mpi=/path/to/MPI/compilers
+    $ make check
+    $ make install   # or sudo make install, if root permissions required
+~~~~
+
+To build netCDF-4 with PnetCDF support, from the top-level netCDF-4 source
+directory, configure netCDF with the "--enable-pnetcdf" option. If PnetCDF
+is built with static library only, add "--disable-shared" option.
```
Comment (edhartnett) [inline docs/install.md]: Why does pnetcdf build with static libraries by default? Aren't shared libraries more performant on HPC systems?

File: docs/install.md (hunk 1 above)
Comment (wkliao) [reply docs/install.md]: It is just a minimum requirement.
I believe when NetCDF incorporated PnetCDF, PnetCDF was 1.6.0 young.

File: docs/install.md (hunk 2 above)
Comment (wkliao) [reply docs/install.md]: Quoted from a web page at NERSC. "Using dynamic libraries may introduce delays in application launch times. This delay is caused by the runtime linking process and the relative inefficiency of symbol lookup in distributed shared objects. Also,  there may be a small performance degradation during execution."
http://www.nersc.gov/users/computational-systems/edison/running-jobs/shared-and-dynamic-libraries/

Comment (edhartnett) [reply docs/install.md]: Interesting. Thanks. I will see if I can test this.

**Pull request merged**